docs/source/tutorial.rst

```diff
@@ -68,10 +68,40 @@ If you want to run only Ansible playbook, you can just run ``ansible-playbook -i
 Step 4: Interact with the LLM Service
 -------------------------------------
 
+You can access the API endpoint by establishing a SSH port-forwarding.
+
 .. note::
 
-   TODO: Fill this out.
+   ansible all -i inventory.ini --ssh-extra-args="-L8080:localhost:80" -m shell -a "sleep infinity"
```

Comment:
Just checking, can we access kind's(docker-engine) NodePort from localhost?

Reply:
Yes, it worked fine with my testing.

Reply:
I could be wrong, but `extraPortMappings` (https://kind.sigs.k8s.io/docs/user/configuration/#extra-port-mappings) is providing the mapping.

https://github.com/llm-operator/llm-operator/blob/main/aws/scripts/setup-kind-cluster.sh#L16

```
ubuntu@ip-172-31-29-15:~$ docker ps
CONTAINER ID   IMAGE                  COMMAND                  CREATED       STATUS         PORTS                                                                 NAMES
c74eba97a815   kindest/node:v1.29.2   "/usr/local/bin/entr…"   6 hours ago   Up 3 minutes                                                                         nvkind-7l7fk-worker
2c8e74526ca4   kindest/node:v1.29.2   "/usr/local/bin/entr…"   6 hours ago   Up 3 minutes   0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 127.0.0.1:33255->6443/tcp   nvkind-7l7fk-control-plane
```